include/swift/IDE/ArgumentCompletion.h

```diff
@@ -52,7 +52,7 @@ class ArgumentTypeCheckCompletionCallback : public TypeCheckCompletionCallback {
     /// Types of variables that were determined in the solution that produced
     /// this result. This in particular includes parameters of closures that
     /// were type-checked with the code completion expression.
-    llvm::SmallDenseMap<const VarDecl *, Type> SolutionSpecificVarTypes;
+    llvm::SmallDenseMap<VarDecl *, Type> SolutionSpecificVarTypes;
```

Comment:
Non-`const` only for `InterfaceTypeRequest{VD}` in `WithSolutionSpecificVarTypesRAII::setInterfaceType`? If so, I would `const_cast` there, just like `ValueDecl::getInterfaceType()` ;)